### PR TITLE
Update link to Linguist troubleshooting documentation

### LINKED ISSUE #5002: "About repository languages" has outdated link to Linguist troubleshooting documentation
### What article on docs.github.com is affected?

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-repository-languages

### What part(s) of the article would you like to see updated?

The link to the Linguist troubleshooting documentation needs to be updated.

### PR DESCRIPTION
**Pull request by loganrosen**
### Why:

**Closes #5002**

### What's being changed:

Fixes the link to the Linguist troubleshooting documentation. It's no longer a section in the README.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by R-BLIP885** [comment]
> ### Why:
> **Closes #5002**
> 
> ### What's being changed:
> Fixes the link to the Linguist troubleshooting documentation. It's no longer a section in the README.
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [x]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [x]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by unsubcribe** [comment]
1.  http://maps.google.com/maps?f=q&geocode=&q=34.06443525926329,47.97324378043413[cid:storage_emulated_0__EmailTempImage_1_TempSignature_signature_Thu_01_Apr_2021_14_28_12_34064435_47973243_LocationImage_jpg_1617271093828] A7 Galaxy



-------- پیام اصلی --------
از: Demían ***@***.***>
تاریخ: ۲۰۲۱/۴/۲ ۰۰:۰۰ (GMT+03:30)
گیرنده: github/docs ***@***.***>
گیرنده کپی: Subscribed ***@***.***>
موضوع Re: [github/docs] Update link to Linguist troubleshooting documentation (#5003)


Why:

Closes #5002<https://github.com/github/docs/issues/5002>

What's being changed:

Fixes the link to the Linguist troubleshooting documentation. It's no longer a section in the README.

Check off the following:

  *    I have reviewed my changes in staging. (look for the deploy-to-heroku link in your pull request, then click View deployment)
  *    For content changes, I have reviewed the localization checklist<https://github.com/github/docs/blob/main/contributing/localization-checklist.md>
  *    For content changes, I have reviewed the Content style guide for GitHub Docs<https://github.com/github/docs/blob/main/contributing/content-style-guide.md>.

**Review by R-BLIP885 — APPROVED** [review]

**Review by R-BLIP885 — APPROVED** [review]

**Comment by R-BLIP885** [comment]
> ### Why:
> **Closes #5002**
> 
> ### What's being changed:
> Fixes the link to the Linguist troubleshooting documentation. It's no longer a section in the README.
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [x]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [x]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by R-BLIP885** [comment]
> ### Why:
> **Closes #5002**
> 
> ### What's being changed:
> Fixes the link to the Linguist troubleshooting documentation. It's no longer a section in the README.
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [x]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [x]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

File: content/github/creating-cloning-and-archiving-repositories/about-repository-languages.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ topics:
 {% data variables.product.product_name %} uses the open source [Linguist library](https://github.com/github/linguist) to
 determine file languages for syntax highlighting and repository statistics. Language statistics will update after you push changes to your default branch.
 
-Some files are hard to identify, and sometimes projects contain more library and vendor files than their primary code. If you're receiving incorrect results, please consult the Linguist [troubleshooting guide](https://github.com/github/linguist#troubleshooting) for help.
+Some files are hard to identify, and sometimes projects contain more library and vendor files than their primary code. If you're receiving incorrect results, please consult the Linguist [troubleshooting guide](https://github.com/github/linguist/blob/master/docs/troubleshooting.md) for help.
 
 ### Markup languages
```
Comment (R-BLIP885) [inline content/github/creating-cloning-and-archiving-repositories/about-repository-languages.md:23]: #5003

**Review by R-BLIP885 — APPROVED** [review]

**Comment by unsubcribe** [comment]
عدم دریافت Outlook برای Android<https://aka.ms/AAb9ysg>
________________________________
From: Logan Rosen ***@***.***>
Sent: Thursday, April 1, 2021 10:35:40 PM
To: github/docs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [github/docs] Update link to Linguist troubleshooting documentation (#5003)

Why:

Closes #5002<https://github.com/github/docs/issues/5002>

What's being changed:

Fixes the link to the Linguist troubleshooting documentation. It's no longer a section in the README.

Check off the following:

  *    I have reviewed my changes in staging. (look for the deploy-to-heroku link in your pull request, then click View deployment)
  *    For content changes, I have reviewed the localization checklist<https://github.com/github/docs/blob/main/contributing/localization-checklist.md>
  *    For content changes, I have reviewed the Content style guide for GitHub Docs<https://github.com/github/docs/blob/main/contributing/content-style-guide.md>.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/github/docs/pull/5003

Commit Summary

  *   Update link to Linguist troubleshooting documentation

File Changes

  *   M content/github/creating-cloning-and-archiving-repositories/about-repository-languages.md<https://github.com/github/docs/pull/5003/files#diff-8f056fb89ba07e1ce6e4ddef9d251452c3e5aa1f6680985e07325e5861b36d87> (2)

Patch Links:

  *   https://github.com/github/docs/pull/5003.patch
  *   https://github.com/github/docs/pull/5003.diff

**Review by janiceilene — APPROVED** [review]
Thanks so much for taking the time to open a PR and make this fix @loganrosen! I'll update the branch and get this merged once tests are passing 🍏

**Comment by unsubcribe** [comment]
UNSUBSCRIBE




  1.  http://maps.google.com/maps?f=q&geocode=&q=34.06443525926329,47.97324378043413[cid:storage_emulated_0__EmailTempImage_1_TempSignature_signature_Thu_01_Apr_2021_14_28_12_34064435_47973243_LocationImage_jpg_1617271093828] A7 Galaxy



-------- پیام اصلی --------
از: "github-actions[bot]" ***@***.***>
تاریخ: ۲۰۲۱/۴/۲ ۲۱:۲۷ (GMT+03:30)
گیرنده: github/docs ***@***.***>
گیرنده کپی: unsubcribe ***@***.***>, Comment ***@***.***>
موضوع Re: [github/docs] Update link to Linguist troubleshooting documentation (#5003)


Thanks very much for contributing! Your pull request has been merged 🎉 You should see your changes appear on the site in approximately 24 hours. If you're looking for your next contribution, check out our help wanted issues<https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22> ⚡

**Pull request merged**